rasa/nlu/utils/spacy_utils.py

```diff
@@ -129,18 +130,120 @@ def get_text(self, example, attribute):
 
         return self.preprocess_text(example.get(attribute))
 
+    @staticmethod
+    def merge_content_lists(freezed_indices, doc_lists):
+        """
```

Comment:
We follow Google docstring format for commenting functions. You can take a look at https://github.com/RasaHQ/rasa/blob/master/rasa/nlu/training_data/training_data.py for the format or set in in your editor(pycharm, etc.) too.